mappings/net/minecraft/server/world/ChunkTicketManager.mapping

```diff
@@ -69,6 +69,7 @@ CLASS net/minecraft/class_3204 net/minecraft/server/world/ChunkTicketManager
 		ARG 4 argument
 	METHOD method_21623 getTicket (J)Ljava/lang/String;
 		ARG 1 pos
+	METHOD method_21683 getDebugString ()Ljava/lang/String;
```

Comment:
I'd suggest `getDebugMessage` or `getDumpMessage` if this is only used for dumping. `getString` doesn't sound very informative...

Reply:
+1 for `getDebugMessage`

Reply:
In this case the string it returns is describing the contents of the tick manager. So really, it's a bit like a `toString` method for dumps. `toDumpString`?

Reply:
```suggestion
	METHOD method_21683 toDumpString ()Ljava/lang/String;
```